src/util/storybook-docs/Layout.jsx

```diff
@@ -17,9 +17,9 @@ const Layout = ({ hasStories = true }) => (
   <>
     <Related />
     <Title />
-    <ImportPath />
     <Subtitle />
     <Description />
+    <ImportPath />
```

Comment:
Because of the way Storybook's Source component looks, I find that it looks more aesthetically pleasing grouped in with the other blocks, so I've moved down the import statement to be below the description.